pkg/api/server/register_containers.go

```diff
@@ -72,6 +72,7 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	//        - name=<name> a container's name
 	//        - network=(<network id> or <network name>)
 	//        - publish=(<port>[/<proto>]|<startport-endport>/[<proto>])
+	//        - pod=(<pod id> or <pod name>
```

Comment:
This is only added for compat but not the libpod endpoint.

Reply:
u are right.  fixed!